src/components/account/AccountFund/AccountFundContent.tsx

```
@@ -61,6 +67,115 @@ export default function AccountFundContent(props: Props) {
 
   const chainConfig = useChainConfig()
 
+  const skipClient = useMemo(
+    () =>
+      new SkipClient({
+        getCosmosSigner: async () => {
+          const offlineSigner = window.keplr?.getOfflineSigner(chainConfig.id.toString())
+          if (!offlineSigner) throw new Error('Keplr not installed')
+          return offlineSigner
+        },
+        getEVMSigner: async (chainID: string) => {
+          const evmWalletClient = (await getWalletClient(config, {
+            chainId: parseInt(chainID),
+          })) as WalletClient
+
+          if (!evmWalletClient) {
+            throw new Error(`getEVMSigner error: no wallet client available for chain ${chainID}`)
+          }
+
+          return evmWalletClient as any
+        },
+      }),
+    [chainConfig],
+  )
+
+  const wrapTransactionPromise = (promise: Promise<void>): Promise<BroadcastResult> => {
+    return promise.then(
+      () => ({ success: true }) as BroadcastResult,
+      (error) => ({ success: false, error }) as BroadcastResult,
+    )
+  }
+  const handleSkipTransfer = useCallback(
+    async (selectedAsset: WrappedBNCoin) => {
+      if (!cosmosAddress || !evmAddress || !selectedAsset) {
+        console.error('Missing required data for transfer')
+        return
+      }
+
+      try {
+        if (!selectedAsset.chain) throw new Error('Chain not found for selected asset')
```

Do you need to throw an error? 
This will trigger the client-side error page on the app.
A console.log error would be enough. An error handling in the UI would be best.